client/app/account/user/advanced/advanced.routing.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash';
+
+angular
+  .module('UserAccount')
+  .config(($stateProvider, $urlRouterProvider, coreConfigProvider) => {
```

Comment:
All the routes were splitted so I suppose it was the compromise made by not creating a module (it would mean create modules for all sub sections splitted)